### PR TITLE
PassiveTotal - limit MD of enrichment cmds

### PR DESCRIPTION
**Pull request by bakatzir**
## Status
- [x] ~~In Progress - pending customer verification~~ Ready

## Print screen:
We have no instance, but an example to the hard limit:
Screen Shot 2020-08-16 at 10 42 12

## Related Issues
fixes: https://github.com/demisto/etc/issues/27394

## Does it break backward compatibility?
   - [x] No

### CODE REVIEW

File: Packs/PassiveTotal/Integrations/integration-PassiveTotal.yml [hunk 1]
```diff
@@ -249,7 +252,12 @@ script:
                     };
                 }
             }
-            var fullResult = [{Type: entryTypes.note, Contents: jsonRes, ContentsFormat: formats.json, HumanReadable: md, EntryContext: context}];
+            if (verboseEnrichment === false) {
+                var fullResult = [{Type: entryTypes.note, Contents: context, ContentsFormat: formats.json, HumanReadable: md, EntryContext: context}];
```
Comment (anara123) [inline Packs/PassiveTotal/Integrations/integration-PassiveTotal.yml:256]: The `context` here contains stuff like DBotScore and Domain. I don't see why we should return it in `Contents`
What exactly is returned in non verboseEnrichment?

Comment (bakatzir) [reply Packs/PassiveTotal/Integrations/integration-PassiveTotal.yml]: Changed the logic to show 20 keys in the md and in the raw-response.

File: Packs/PassiveTotal/Integrations/integration-PassiveTotal.yml [hunk 2]
```diff
@@ -223,6 +217,17 @@ script:
                     }
                 }
             }
+            jsonKeys = Object.keys(jsonRes);
+            if (jsonKeys.length > 20) {
+                limit = 20;
+            } else {
+                limit = jsonKeys.length;
+            }
+            var jsonMd = {};
```
Comment (yuvalbenshalom) [inline Packs/PassiveTotal/Integrations/integration-PassiveTotal.yml:226]: just to make sure, we only need to filter the MD? the context itself is not packed?
dose this make sense?

Comment (bakatzir) [reply Packs/PassiveTotal/Integrations/integration-PassiveTotal.yml:226]: The related domains do not go into the context.

File: Packs/PassiveTotal/ReleaseNotes/1_0_1.md [hunk 3]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### PassiveTotal
+- Added a limit to the number of related domain showed in the war room for enrichment commands.
```
Comment (amshamah419) [inline Packs/PassiveTotal/ReleaseNotes/1_0_1.md:4]:
```suggestion
- Added a limit to the number of related domains showed in the war room for enrichment commands.
```

**Pull request merged**